### PR TITLE
Update bosses

### PR DESCRIPTION
**Pull request by JaCait**

### CODE REVIEW

File: roster/urls.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
 from django.urls import path
-
 from . import views
+
+urlpatterns = [
+    path('Bosses', views.Bosses),
```
Comment (RioluM) [inline roster/urls.py:5]: Path should not be uppercase

File: roster/views.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
 from django.shortcuts import render
 
-# Create your views here.
+def Bosses(request):
```
Comment (RioluM) [inline roster/views.py:3]: view method should not be uppercase

**Review by RioluM — CHANGES_REQUESTED** [review]

**Review by JaCait — COMMENTED** [review]
fixed

File: roster/views.py [hunk 3]
```diff
@@ -1,3 +1,8 @@
 from django.shortcuts import render
 
-# Create your views here.
+def bosses(request):
+    bossesFile = open('./bosses.txt', 'r')
```
Comment (RioluM) [inline roster/views.py:4]: After using a file, remember to close it

**Review by RioluM — CHANGES_REQUESTED** [review]

**Review by RioluM — APPROVED** [review]

**Pull request merged**